manifests/a/Avanquestpdfforge/PDFCreator-Terminal-Server/5.2.0/Avanquestpdfforge.PDFCreator-Terminal-Server.installer.yaml

```diff
@@ -0,0 +1,22 @@
+PackageIdentifier: Avanquestpdfforge.PDFCreator-Terminal-Server
+PackageVersion: 5.2.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+InstallerSwitches:
+  Silent: /COMPONENTS="MAINPROGRAM" /SILENT /DEFERLICENSECHECK
```

Comment:
Which license check is deferred with the switch `DEFERLICENSECHECK` ("Defer License Check")?

Reply:
Hi @stephengillie,

I am a colleague of @frank-heindoerfer and part of the development team of PDFCreator. I will try to provide you with some more information.

The `DEFERLICENSECHECK` parameter is used to internally postpone the license check and activation to the first start of the software (the user is prompted to enter a license key on first start). This allows the software to be installed without providing a license key to the installer. 

The parameter is necessary here in order to allow our licensed software to be installed with a simple `winget install` command (without additional parameters).

Reply:
Hello @stephengillie,

@karlo-mirkovic  has explained the function /DEFERLICENSECHECK very well and I have nothing to add.

>The DEFERLICENSECHECK parameter is used to internally postpone the license check and activation to the first start of the software (the user is prompted to enter a license key on first start). This allows the software to be installed without providing a license key to the installer.

>The parameter is necessary here in order to allow our licensed software to be installed with a simple winget install command (without additional parameters).